README.md

```diff
@@ -83,26 +85,34 @@ Usage of ./endlessh-go
         comma-separated list of pattern=N settings for file-filtered logging
 ```
 
+## Using privileged ports (<1024)
+
+If you want to run the image with privileged ports (below 1025), you need to set the container user to root:
+
+```yml
+user: root
+```
+
 ## Metrics
 
 Endlessh-go exports the following Prometheus metrics.
 
-| Metric                               | Type  | Description  |
-|--------------------------------------|-------|--------------|
-| endlessh_client_open_count_total     | count | Total number of clients that tried to connect to this host. |
-| endlessh_client_closed_count_total   | count | Total number of clients that stopped connecting to this host. |
-| endlessh_sent_bytes_total            | count | Total bytes sent to clients that tried to connect to this host. |
-| endlessh_trapped_time_seconds_total  | count | Total seconds clients spent on endlessh. |
-| endlessh_client_open_count           | count | Number of connections of clients. <br> Labels: <br> <ul><li> `ip`: IP of the client </li> <li>  `country`: Country of the IP </li> <li>  `location`: Country, Region, and City </li> <li>  `geohash`: Geohash of the location </li></ul> |
-| endlessh_client_trapped_time_seconds | count | Seconds a client spends on endlessh. <br> Labels: <br> <ul><li>  `ip`: IP of the client </li></ul> |
+| Metric                               | Type  | Description                                                                                                                                                                                                                           |
```

Comment:
Which editor are you using?

I don't thin we need the extra space at the end to align each line.